core/pkg/server/config_test.go

```diff
@@ -0,0 +1,69 @@
+package server
```

Comment:
what is your opinion of having it server_test?
i see that you don't any private functions anyways

Reply:
updated! i read in the Go docs that tests are usually in the same package as the tested code, but I'm fine with using the `_test` suffix for consistency

Reply:
we can agree on one approach and stick to it. The advantage for using a _test is that it forces us to think about our apis, but maybe for internal function we could use the same package?